### PR TITLE
Halfway decent fix to allow changing of minimap size and zoom midgame 

### PR DESCRIPTION
**Pull request by JMM889901**
it errr does the thing and errrr minimap get bigger :clueless:

### CODE REVIEW

File: Northstar.Client/mod/scripts/vscripts/client/cl_minimap.gnut [hunk 1]
```diff
@@ -0,0 +1,950 @@
+global function ClMinimap_Init
+
+global function ClientCodeCallback_MinimapEntitySpawned
+
+global function DumbMinimapFunction
```
Comment (BobTheBob9) [inline Northstar.Client/mod/scripts/vscripts/client/cl_minimap.gnut:5]: maybe remove this lol

**Review by BobTheBob9 — CHANGES_REQUESTED** [review]

File: Northstar.Client/mod/scripts/vscripts/client/cl_minimap.gnut (hunk 1 above)
Comment (JMM889901) [reply Northstar.Client/mod/scripts/vscripts/client/cl_minimap.gnut]: Forgor

**Comment by JMM889901** [comment]
No merge? image

**Review by uniboi — APPROVED** [review]
image
works in testing

**Pull request merged**